pandas/core/indexes/multi.py

```diff
@@ -1412,6 +1420,77 @@ def from_product(cls, iterables, sortorder=None, names=None):
         labels = cartesian_product(labels)
         return MultiIndex(levels, labels, sortorder=sortorder, names=names)
 
+    @classmethod
+    def from_frame(cls, df, squeeze=True, names=None):
+        """
+        Make a MultiIndex from a DataFrame.
+
```

Comment:
add a versionadded tag

Reply:
Found the version from the pending what's new page. Added the tag for 0.24.0, and updated the what's new page.